packages/server-core/src/media/static-resource/static-resource.resolvers.ts

```diff
@@ -82,17 +82,10 @@ export const staticResourceResolver = resolve<StaticResourceType, HookContext>(
     thumbnailURL: virtual(async (staticResource, context) => {
       if (!staticResource.thumbnailKey) return
       const storageProvider = getStorageProvider()
-      /** @todo optimize this */
-      const thumbnailStaticResource = await context.app.service('static-resource').find({
-        query: {
-          key: staticResource.thumbnailKey
-        }
-      })
-      if (!thumbnailStaticResource.data.length) return
       return (
         storageProvider.getCachedURL(staticResource.thumbnailKey, context.params.isInternal) +
         '?hash=' +
-        thumbnailStaticResource.data[0].hash.slice(0, 6)
+        staticResource.hash.slice(0, 6)
```

Comment:
why are we adding the hash for the static resource to the static resource's thumbnail? this will invalidate cache incorrectly for the thumbnail

Reply:
I'm removing this change. I don't think it's related to this PR. Basically I believe the CMS branch broke this by not migrating any of the older thumbnails into static resources. I don't have a leg in the putting thumbnails into static resource conversation as I am very much new to this conversation.

Reply:
The resources.json is now the source of truth and needs to be updated accordingly.

Reply:
thumbnails are not static resources and shouldn't be in resources.json